docs/setup/jetson.md

```diff
@@ -0,0 +1,162 @@
+<!--
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is there a reason that using BSD 3-clause as with the rest of this repository is not acceptable? If NVIDIA's contributions must be Apache 2.0, then we will also need to update LICENSE.txt with exceptions, and that will need to be kept up to date. I would prefer if possible to keep the same license for the whole project.

Reply:
Copyright notice changed to BSD-3.
Resolved in commit 2cfff17